src/frontend/src/lib/components/tokens/TokensZeroBalance.svelte

```diff
@@ -4,7 +4,7 @@
 </script>
 
 <div class="content">
-	<span class="block text-sm text-blue-ribbon">{$i18n.tokens.text.balance}</span>
+	<span class="block text-sm text-foreground-brand-primary">test{$i18n.tokens.text.balance}</span>
```

Comment:
here there is an intruder? eheh string `test` ?

Reply:
thanks for spotting this